LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright 2024 Odis Foundation or its affiliates. All Rights Reserved.
+Copyright 2024 Valkey Foundation or its affiliates. All Rights Reserved.
```

Comment:
I think this should be Linux foundation, they will own the trademark.

Reply:
https://github.com/valkey-io/valkey-website/pull/1/commits/cc2c3fea23c1f603fdf9192582ef4ff2b735f0fe changes the 3 references to the foundation.